internal/bundle/inspect/inspect.go

```diff
@@ -199,3 +209,55 @@ func (bi *Info) getBundleDataWasmAndSignatures(name string) error {
 
 	return nil
 }
+
+func fileInfoForRegoVersion(regoVersion ast.RegoVersion, path string, includeAnnotations bool) (*Info, error) {
+	res, err := loader.NewFileLoader().
+		WithRegoVersion(regoVersion).
+		WithSkipBundleVerification(true).
+		WithProcessAnnotation(true). // Always process annotations, for enriching namespace listing
+		WithJSONOptions(&json.Options{
+			MarshalOptions: json.MarshalOptions{
+				IncludeLocation: json.NodeToggle{
+					// Annotation location data is only included if includeAnnotations is set
+					AnnotationsRef: includeAnnotations,
+				},
+			},
+		}).
+		All([]string{path})
+	if err != nil {
+		return nil, err
+	}
+	bi := &Info{
+		Namespaces: make(map[string][]string, len(res.Modules)),
+	}
+
+	moduleMap := make(map[string]*ast.Module, len(res.Modules))
+
+	for _, m := range res.Modules {
+		bi.Namespaces[m.Parsed.Package.Path.String()] = append(
+			bi.Namespaces[m.Parsed.Package.Path.String()],
+			filepath.Clean(m.Name),
+		)
+		moduleMap[m.Name] = m.Parsed
+	}
+
+	if includeAnnotations {
+		as, errs := ast.BuildAnnotationSet(maps.Values(moduleMap))
```

Comment:
Did we add an extra dependency to OPA just for `maps.Values`? I think we could just add a util function for that if we don't have one already.